src/app/public/styles/themes/modern/_buttons.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../variables';
+@import '../../variables-public';
+@import './_compat/mixins';
+
+.sky-theme-modern {
+  .sky-btn {
+    font-size: $sky-theme-modern-font-body-default-size;
+  }
+
+  .sky-btn-tab {
+    @include sky-theme-modern-btn-tab;
+  }
+
+  .sky-btn-tab-selected:not(.sky-btn-tab-disabled) {
+    @include sky-theme-modern-btn-tab-selected;
+  }
+
+  .sky-btn-tab:hover:not(.sky-btn-tab-disabled) {
+    background-color: transparent;
```

Comment:
With current tabs, it looks like there's a hover background color of light grey. Is that changing?

Reply:
@Blackbaud-ToddRoberts any opinion on this? The spec doesn't show a hover state.